### PR TITLE
(WIP) (MAINT) Appveyor New Cloud Test only 

### PR DESCRIPTION
**Pull request by jcoconnor**
This is purely a test of the new Appveyor test/build system.
NOT FOR MERGE !!!!

Associated with WIN-101 Epic.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -10,16 +10,11 @@ init:
...
-    RUBY_VER: 24
-  - PUPPET_GEM_VERSION: ~> 5.0
-    RUBY_VER: 24-x64
-  - PUPPET_GEM_VERSION: 4.7.1
-    RUBY_VER: 21-x64
+  - APPVEYOR_BUILD_WORKER_CLOUD: puppet-appveyor-test
+    PUPPET_GEM_VERSION: ~> 5.0
+    APPVEYOR_BUILD_WORKER_IMAGE: 2012R2 PS4
+    TEST_VARIABLE: 42
+    RUBY_VERSION: 24
```
Comment (Iristyle) [inline appveyor.yml]: Doesn't look like it picked up Ruby 2.4... I think there's some magic baked into the AppVeyor default images that use this environment variable to set current ruby version via URU or similar.

Comment (Iristyle) [reply appveyor.yml]: The gem install failure is for this gem - https://rubygems.org/search?utf8=%E2%9C%93&query=puppet-module-win-default

Note that only 2.1, 2.3 and 2.4 versions are published.

Comment (glennsarti) [reply appveyor.yml]: Not really magic.  First line in the file
https://github.com/puppetlabs/puppetlabs-powershell/blob/master/appveyor.yml#L25-L26

Note `RUBY_VER` has been renamed `RUBY_VERSION`

Comment (jcoconnor) [reply appveyor.yml]: Thanks @glennsarti That worked a treat - and checks pass.

**Comment by jcoconnor** [comment]
Closing - will pull out of the ether if we ever get back to this.

**Pull request closed without merging**